tests/suites/test_suite_pkcs11_client.data

```diff
@@ -5,3 +5,15 @@ pk_import_sign:"data_files/server1.key"
 PKCS#11 RSA generate and sign
 depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
 pk_generate_sign:MBEDTLS_PK_RSA
+
+PKCS#11 RSA import, sign and verify with Cryptoki
+depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
+pk_import_sign_verify:"data_files/server1.key"
+
+PKCS#11 RSA import, sign with MbedTLS and verify with Cryptoki
+depends_on:MBEDTLS_PK_C:MBEDTLS_RSA_C
+pk_import_verify_signed:"data_files/server1.key"
+
+PKCS#11 RSA verify a hardcoded signature with Cryptoki
```

Comment:
Can we add a test here for signing with Cryptoki and verifying with Mbed TLS?

Reply:
Ah, it seems that `pk_import_sign` does already do that.

Reply:
Most tests should be of the first variety, since the point of using opaque keys is that they might not be extractable.

Tests of the second variety are specifically to test the import functionality.

Reply:
For public keys, the second variety is important, though - isn't it?

Reply:
The focus of this project is on support for non-extractable private keys. We do round off the corner and support opaque public keys as well to some extent, but we're not trying to cover all use cases (and in particular they aren't supported in TLS). Since the focus is on non-extractable keys, we don't expose an export functionality in the API.

Even for public keys, we do test `{import; software_sign; pkcs11_verify;}`.